extensions/amp-autocomplete/0.1/amp-autocomplete.js

```diff
@@ -354,13 +401,19 @@ export class AmpAutocomplete extends AMP.BaseElement {
   }
 
   /**
-   * Returns true if the given element is a suggested item.
+   * Returns the nearest ancestor element that is a suggested item.
    * @param {?Element|?EventTarget} element
+   * @return {?Element}
    * @private
    */
-  isItemElement_(element) {
-    return element !== null &&
-      element.classList.contains('i-amphtml-autocomplete-item');
+  getItemElement_(element) {
+    if (element === null) {
+      return null;
+    }
+    if (element.classList.contains('i-amphtml-autocomplete-item')) {
+      return /**@type {!Element} */ (element);
```

Comment:
It was because I annotated the return type as `Element` and sometimes it returns the `EventTarget` that is passed in. However, since `selectItem_` takes either type and it's the only place `getItemElement_` is called, I just updated the annotation to include both types.